plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/core-profiler.spec.js

```diff
@@ -122,7 +122,7 @@ test.describe( 'Store owner can complete the core profiler', () => {
 			).toBeVisible();
 
 			// go to the plugins page to make sure that extensions weren't installed
-			await page.goto( 'wp-admin/plugins.php' );
+			await page.goto( 'wp-admin/plugins.php?plugin_status=active' );
```

Comment:
In some envs, some plugins are pre-installed. To be sure, we're checking the page with active plugins.